Automattic/wp-calypso
# A4A: Add support contact form

Related to https://github.com/Automattic/jetpack-roadmap/issues/1368

## Proposed Changes
It was decided to eliminate zendesk chat with simple contact support form which would rely on API endpoint on wpcom.

| Pressable | Others |
| ---- | ---- |
| <img width="1344" alt="Screenshot 2024-04-02 at 3 57 30 PM" src="https://github.com/Automattic/jetpack-roadmap/assets/60262784/a53043c3-df9f-4257-be22-79ed9f919d48"> | <img width="1331" alt="Screenshot 2024-04-02 at 3 56 42 PM" src="https://github.com/Automattic/jetpack-roadmap/assets/60262784/13131da9-86f6-48bf-b044-8a19c66f5647"> |

## Testing Instructions
- Easiest way to test fully is to follow instructions of this diff: D144258-code
- To test view just visit a live link and press the Contact Support button on the overview page.
- You can after check `#a4a-support` slack channel to see that issue is created.

## Pre-merge Checklist
